### DISCUSSION TITLE
I am getting an error from the library related to MatIconButton

### DISCUSSION THREAD

**Discussion by BagalBalasaheb** [angular-material-extensions/password-strength#402] [Q&A]
Error ----->

./node_modules/@angular-material-extensions/password-strength/fesm2022/angular-material-extensions-password-strength.mjs:434:19-35 - Error: export 'MatIconButton' (imported as 'i1') was not found in '@angular/material/button' (possible exports: MatAnchor, MatButton, MatButtonModule)

**Comment by AnthonyNahas** [comment]
which version are you using?

**Reply by sharmaniki11** [reply]
Below are the version I am using:
"@angular-material-extensions/password-strength": "^16.0.0"
Angular: 16.2.12

**Reply by sharmaniki11** [reply]
I update the angular material version to latest one and the error is gone.
Thank you @AnthonyNahas for the reply
